packages/dashboard/src/components/tasks/tasks-app.tsx

```diff
@@ -245,23 +246,42 @@ export const TasksApp = React.memo(
           return [];
         }
 
-        const resp = await rmf.tasksApi.queryTaskStatesTasksGet(
-          undefined,
-          undefined,
-          undefined,
-          undefined,
-          undefined,
-          undefined,
-          undefined,
-          `0,${timestamp.getTime()}`,
-          undefined,
-          undefined,
-          undefined,
-          undefined,
-          '-unix_millis_start_time',
-          undefined,
-        );
-        const allTasks = resp.data as TaskState[];
+        const taskStateCount = (
+          await rmf.tasksApi.taskStatesCountTasksCountGet(
+            undefined,
+            undefined,
+            undefined,
+            undefined,
+            undefined,
+            undefined,
+            undefined,
+            undefined,
+            `0,${timestamp.getTime()}`,
```

Comment:
Do we need to get every tasks? Can we limit it to the last X days?

Reply:
yeah it makes sense, I'll start off with 31 days, which should satisfy the requirements.

Reply:
https://github.com/open-rmf/rmf-web/pull/909/commits/1de251a91f11ab970c2f114b16c69fe514ff7b63